### PR TITLE
fix: audit seedlot working great

### LINKED ISSUE #927: Fix Audit Seedlot database function
Fix `Seedlot` Audit function not working properly and logging errors:

```sh
2024-03-25 21:52:09.000 UTC [326279] WARNING: [AUDIT.IF_MODIFIED_FUNC] - UDF ERROR [OTHER] - SQLSTATE: 42601, SQLERRM: INSERT has more expressions than target columns
282024-03-25 21:52:09.000 UTC [326279] CONTEXT: PL/pgSQL function seedlot_if_modified_func() line 215 at RAISE
292024-03-25 21:52:11.201 UTC [326279] WARNING: [AUDIT.IF_MODIFIED_FUNC] - UDF ERROR [OTHER] - SQLSTATE: 42601, SQLERRM: INSERT has more expressions than target columns
302024-03-25 21:52:11.201 UTC [326279] CONTEXT: PL/pgSQL function seedlot_if_modified_func() line 215 at RAISE
312024-03-25 21:52:11.206 UTC [326279] WARNING: [AUDIT.IF_MODIFIED_FUNC] - UDF ERROR [OTHER] - SQLSTATE: 42601, SQLERRM: INSERT has more expressions than target columns
322024-03-25 21:52:11.206 UTC [326279] CONTEXT: PL/pgSQL function seedlot_if_modified_func() line 215 at RAISE
```

### PR DESCRIPTION
**Pull request by RMCampos**
# Description
Closes #927 

#### New
- Migration SQL script fixing wrong columns

#### Changed
- None

#### Removed
- None

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests

###  What gif/image best describes this PR or how it makes you feel?

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-31-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-31-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-31-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

---
Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Frontend](https://nr-spar-31-frontend.apps.silver.devops.gov.bc.ca/)
[Backend](https://nr-spar-931-backend.apps.silver.devops.gov.bc.ca/actuator/health)
[Oracle-API](https://nr-spar-931-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge-main.yml)

### CODE REVIEW

File: backend/src/main/resources/db/migration/V28__fix_seedlot_audit_trigger.sql [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    /*entry_userid                   */ NEW.entry_userid,
+    /*entry_timestamp                */ NEW.entry_timestamp,
+    /*update_userid                  */ NEW.update_userid,
+    /*update_timestamp               */ NEW.update_timestamp,
+    /*revision_count                 */ NEW.revision_count
+    );
+        RETURN NEW;
+    elsif (TG_OP = 'DELETE') then
+        v_old_data := ROW(OLD.*);
+        /*nsert into spar.logged_actions (schema_name,table_name,user_name,action,original_data,query)
```
Comment (mgaseta) [inline backend/src/main/resources/db/migration/V28__fix_seedlot_audit_trigger.sql:78]: Typo: `Insert`

**Review by mgaseta — APPROVED** [review]
LGTM! Only one typo to fix!
Question: is it ok to leave the fields regarding lat/long/elev out of the audit table?

**Review by DerekRoberts — APPROVED** [review]
Hooray!  Very good.  :)

**Review by craigyu — APPROVED** [review]

**Comment by RMCampos** [comment]
> LGTM! Only one typo to fix! Question: is it ok to leave the fields regarding lat/long/elev out of the audit table?

Great catch @mgaseta adding them now. Thanks 🙏🏻 🚀

**Pull request merged**